fiftyone/utils/cvat.py

```diff
@@ -3903,7 +3903,13 @@ def get_project_tasks(self, project_id):
             the list of task IDs
         """
         resp = self.get(self.project_url(project_id)).json()
-        return [task["id"] for task in resp.get("tasks", [])]
+        tasks = []
+        for task in resp.get("tasks", []):
+            if isinstance(task, int):
```

Comment:
An alternative here would be to condition the logic explicitly on the CVAT server version. At min, I'd suggest documenting why `tasks` could have different contents here, for our future sanity.